src/drivers/infiniband/hermon.c

```diff
@@ -2097,7 +2097,7 @@ static void hermon_destroy_eq ( struct hermon *hermon ) {
  * @v eqe		Port state change event queue entry
  */
 static void hermon_event_port_state_change ( struct hermon *hermon,
-					     union hermonprm_event_entry *eqe){
+					     union hermonprm_event_entry *eqe ) {
```

Comment:
This pushes the line length over 80 characters, sorry.  There are several other places where I use this to avoid overlength lines, e.g. in `hermon_cmd_sw2hw_cq()`

Reply:
That's.. one way to do it ;-)

Patch reworked.